awx/main/access.py

```diff
@@ -1325,7 +1325,7 @@ class ExecutionEnvironmentAccess(BaseAccess):
 
     def filtered_queryset(self):
         return ExecutionEnvironment.objects.filter(
-            Q(organization__in=Organization.accessible_pk_qs(self.user, 'execution_environment_admin_role')) |
+            Q(organization__in=Organization.accessible_pk_qs(self.user, 'member_role')) |
             Q(organization__isnull=True)
         ).distinct()
 
```

Comment:
Leaving this here, since `can_add` is outside of the commentable lines of code -- it turns out that `check_related` takes an optional `mandatory` flag that I think does what we want wrt Org Admins / EE Admins not being able to create global EEs.